cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/edges/Edge.kt

```diff
@@ -73,6 +74,13 @@ abstract class Edge<NodeType : Node> : Persistable, Cloneable {
 
     abstract var labels: Set<String>
 
+    /**
+     * Whether the Edge is pointing to or from the overlay network is defined by its connected
```

Comment:
I'd change this description to something like "`true` if one of the connected nodes is an overlay node".

Reply:
also `Edge` should be `[Edge]`

Reply:
Done